docs/contributing/how-to-release.md

```diff
@@ -182,22 +192,28 @@ Submit to https://isocpp.org/blog/suggest (need to be logged in to do it).
 
 For major releases, submit the announcement to https://slashdot.org/submission
 
-## Version Updates
+## Post-Release Steps
 
-Trac: mark the milestone corresponding to the release as completed and add a
+* Trac: mark the milestone corresponding to the release as completed and add a
```

Comment:
The change of `-` to `*` here and to `+` below seems weird, was this intentional?